theme/styles/snippets/add-to-cart-container.scss

```diff
@@ -0,0 +1,29 @@
+.AddToCartContainer {
+  &__select-wrapper {
+    overflow: hidden;
+    position: relative;
+    
+      &::after {
+        content: "▼";
```

Comment:
Using this unicode character, but will ask XXIX if they prefer the exact SVG from the Figma file